backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java

```diff
@@ -570,4 +620,77 @@ private static void deleteEventType(String eventTypeId, boolean expectedResult)
 
         assertEquals(expectedResult, result);
     }
+
+    private static BehaviorGroup buildDefaultBehaviorGroup(String displayName, String bundleId) {
+        BehaviorGroup bg = new BehaviorGroup();
+        bg.setDisplayName(displayName);
+        bg.setBundleId(UUID.fromString(bundleId));
+        return bg;
+    }
+
+    private static String createDefaultBehaviorGroup(String displayName, String bundleId) {
+        BehaviorGroup behaviorGroup = buildDefaultBehaviorGroup(displayName, bundleId);
+
+        return given()
+                .basePath(API_INTERNAL)
+                .contentType(JSON)
+                .body(Json.encode(behaviorGroup))
+                .post("/behaviorGroups/default")
+                .then()
+                .statusCode(200)
+                .contentType(JSON)
+                .extract()
+                .body()
+                .jsonPath()
+                .getString("id");
+    }
+
+    private static void updateDefaultBehaviorGroup(String behaviorGroupId, String displayName, String bundleId, boolean expectedResult) {
+        BehaviorGroup behaviorGroup = buildDefaultBehaviorGroup(displayName, bundleId);
+
+        Boolean result = given()
+                .basePath(API_INTERNAL)
+                .contentType(JSON)
+                .body(Json.encode(behaviorGroup))
+                .pathParam("behavior_group_id", behaviorGroupId)
+                .put("/behaviorGroups/default/{behavior_group_id}")
```

Comment:
We could use a consistent path between runtime and test to make searches easier:
```suggestion
                .pathParam("id", behaviorGroupId)
                .put("/behaviorGroups/default/{id}")
```